enumerable.rb

```diff
@@ -122,8 +124,12 @@ def my_inject(*obj)
   end
 end
```

Comment:
- **_`#my_inject` raises a "LocalJumpError" when no block or argument is given_**